.travis/travis-checks-js.sh

```diff
@@ -19,11 +19,6 @@ if [ "$CHECK_CORRECTNESS" = true ] ; then
 fi
 
 if [ "$CHECK_TESTS" = true ] ; then
-  # build the Gutenberg packages
-  pushd gutenberg
-  npm i || pFail
-  popd
-
```

Comment:
Please remember to remove the steps from README.md as well

Reply:
Good catch! Will do it now.